doc/calibrations.rst

```diff
@@ -0,0 +1,56 @@
+.. highlight:: rest
+
+============
+Calibrations
+============
+
+This document gives an overview of the calibration
+steps of PYPIT.  There is additional documentation
+for several of the more complex steps.
+
+Overview
+========
+
+Here is the sequence of events:
+
+========== ============= ===========================================
+Step       Products      Description
+========== ============= ===========================================
+ampsec     datasec image 2D image describing the detectoe pixels for analysis
```

Comment:
armlsd.py still has 'get_ampsec_trimmed' at the start of its procedure.
Looking at the method, we should probably rename it.

in fact, I just did.